lib/rmt/scc_sync.rb

```diff
@@ -1,13 +1,27 @@
 require 'suse/connect/api'
 require 'rmt/config'
+require 'rmt/cli'
 
-class RMT::SCCSync
+# rubocop:disable Rails/Exit
+# rubocop:disable Rails/Output
 
-  def initialize(logger = nil)
-    @logger = logger || Logger.new(nil)
+class RMT::SCCSync < RMT::CLI
+
+  def initialize(args = [], local_options = {}, config = {})
+    super
+    @logger = Logger.new(STDOUT)
+    @logger.level = options[:verbose] ? 0 : 1
   end
 
+  class_option :verbose, aliases: '-v', type: :boolean
+
+  desc 'sync', 'Synchronize database with SCC'
   def sync
+    unless (Settings.scc.username && Settings.scc.password)
+      puts 'SCC credentials not set.'
+      exit 1
```

Comment:
Maybe let's subclass an exception, raise it here, catch it in `bin/rmt-repos` and `exit` there?
There's already similar code in `rmt-mirror`:

```ruby
  rescue RMT::Mirror::Exception => e
    warn e.to_s
    exit 1
  end
```